WeScan/Extensions/CGSize+Utils.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CGSize+Utils.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 17/2/2019.
+//  Copyright © 2019 WeTransfer. All rights reserved.
+//
+
+import Foundation
+
+extension CGSize {
+    func scaleFactor(forMaxWidth maxWidth: CGFloat, maxHeight: CGFloat) -> CGFloat {
```

Comment:
We should test this function

Reply:
I've added documentation and tests for it 🙂